aggregator_api/src/lib.rs

```diff
@@ -170,14 +172,21 @@ async fn post_task<C: Clock>(
                 Status::BadRequest,
             )
         })?;
-        Vec::from([AuthenticationToken::try_from(token_bytes).map_err(|_| {
-            Error::new(
-                "Invalid HTTP header value in aggregator_auth_token".to_string(),
-                Status::BadRequest,
-            )
-        })?])
+        Vec::from([
+            // TODO(#472): Each token in the PostTaskReq should indicate whether it is a bearer
+            // token or a DAP-Auth-Token. For now, assume the latter.
```

Comment:
Not sure I follow--does this mean that with this PR, all Task POSTs will record themselves as `DapAuth` tokens regardless of them being `Bearer` or `DapAuth` token?

It smells like this would blow up if someone attempts to use a bearer token and has it recorded as the wrong type in the DB.

Reply:
This is just for the "aggregator API" that the control plane uses to set up new tasks. The control plane does not yet have a concept of different types of authentication tokens, so this won't lead to any surprises.